compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala

```diff
@@ -271,13 +272,12 @@ case class ApplicationCompiler(typeAliases: Map[String, Type],
         case (name, ExecutableLink(_, _, _, applet: DxApplet))     => (name, applet.id)
         case (name, ExecutableLink(_, _, _, workflow: DxWorkflow)) => (name, workflow.id)
       }
-      .map {
-        case (name, id) =>
-          JsObject(
-              Constants.BundledDependsNameKey -> JsString(name),
-              Constants.BundledDependsIdKey -> JsObject(DxUtils.DxLinkKey -> JsString(id)),
-              Constants.BundledDependsStagesKey -> JsArray(Vector.empty)
-          )
+      .map { case (name, id) =>
```

Comment:
we can use `newlines.beforeCurlyLambdaParams = multilineWithCaseOnly` to keep previous format @jdidion